src/org/zaproxy/zap/view/widgets/WritableFileChooser.java

```diff
@@ -24,6 +24,10 @@ public WritableFileChooser(File currentDirectory) {
 	@Override
 	public void approveSelection() {
 		File selectedFile = getSelectedFile();
+		//Warns if there is less than 5MB available
+		if (selectedFile.getUsableSpace() < 5000000) {
```

Comment:
If the file does not exist getUsableSpace() returns 0.

5000000 could/should be extracted to a constant, in which case the comment could be removed (if the code is clearer).